src/applications/mhv-medical-records/components/CareSummaries/AdmissionAndDischargeDetails.jsx

```diff
@@ -142,31 +142,40 @@ ${record.summary}`;
         <h3 className="vads-u-font-size--base vads-u-font-family--sans">
           Location
         </h3>
-        <p data-testid="note-record-location"> {record.location}</p>
+        <p data-testid="note-record-location" data-dd-privacy="mask">
+          {' '}
+          {record.location}
+        </p>
         {record.sortByField !== dischargeSummarySortFields.ADMISSION_DATE &&
           record.sortByField !== null && (
             <>
               <h3 className="vads-u-font-size--base vads-u-font-family--sans">
                 Date admitted
               </h3>
-              <p data-testid="note-admission-date">{record.admissionDate}</p>
+              <p data-testid="note-admission-date" data-dd-privacy="mask">
+                {record.admissionDate}
+              </p>
             </>
           )}
         {record.sortByField !== dischargeSummarySortFields.DISCHARGE_DATE && (
           <>
             <h3 className="vads-u-font-size--base vads-u-font-family--sans">
               Date discharged
             </h3>
-            <p data-testid="note-discharge-date">{record.dischargeDate}</p>
+            <p data-testid="note-discharge-date" data-dd-privacy="mask">
+              {record.dischargeDate}
+            </p>
           </>
         )}
         <h3 className="vads-u-font-size--base vads-u-font-family--sans">
           Discharged by
         </h3>
-        <p data-testid="note-discharged-by">{record.dischargedBy}</p>
+        <p data-testid="note-discharged-by" data-dd-privacy="mask">
+          {record.dischargedBy}
+        </p>
       </div>
 
-      <div className="test-results-container">
+      <div className="test-results-container" data-dd-privacy="mask">
         <h2>Summary</h2>
         <p
           data-testid="note-summary"
```

Comment:
Let's put the mask as tight as we can around the sensitive data. So here, you can put it on the `<p>` tag. That way we can still see "Summary" in the session playback.

Reply:
fixed